lib/babel_compile.js

```diff
@@ -1,23 +1,17 @@
 var babel = require("babel-core");
 var assign = require('object.assign');
 
-var mapPlugin = {
-	'commonjs': 'babel-plugin-transform-es2015-modules-commonjs',
-	'amd': 'babel-plugin-transform-es2015-modules-amd'
-};
-
 module.exports = function(source, compileOptions, options){
 	var opts = assign({}, options.babelOptions, {
 		sourceMap: compileOptions.sourceMaps || false
 	});
 
 	opts.presets = opts.presets || [
-		require("babel-preset-es2015-no-commonjs"),
-		require("babel-preset-react"),
-		require("babel-preset-stage-0")
+		["es2015", {"loose": false, "modules": compileOptions.modules}],
+		"babel-preset-react",
+		"babel-preset-stage-0"
```

Comment:
We do need to require these, otherwise it will try to find them in the user's own node_modules, but they are actually dependencies of transpile